.github/workflows/misc-sync-templates.yml

```diff
@@ -37,6 +96,18 @@ jobs:
     env:
       template-path: "polkadot-sdk-${{ matrix.template }}-template"
     steps:
+      # 0. download dev chain spec
+      - uses: actions/checkout@v4
+        with:
+          path: polkadot-sdk
+          ref: "${{ github.event.inputs.stable_release_branch }}"
+
+      - name: Download template artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: artifacts-${{ matrix.template }}
+          path: templates/${{ matrix.template }}/
+        if: matrix.template != 'solochain'
```

Comment:
Can we remove the first action checkout here? We're doing it once more below, after setting the git identity. I am thinking we can also move the `Download template artifacts` below the actions checkout that runs after `Configure git identity`. What do you think?